les/state_accessor.go

```diff
@@ -53,9 +53,10 @@ func (leth *LightEthereum) stateAtTransaction(ctx context.Context, block *types.
 	}
 	// Recompute transactions up to the target index.
 	signer := types.MakeSigner(leth.blockchain.Config(), block.Number())
+	l1FeeContext := core.NewL1FeeContext(leth.blockchain.Config(), statedb)
```

Comment:
What do you think about passing `nil` in the L1 tests to reduce diff?

Reply:
I tried making the `baseFee` and `l1Cost` a slice of params, so that `l1Cost` is optional, as per @protolambda suggestion below. This reduces the diff even more, since we don't even need to pass `nil`, at the cost of additional slice checks below.